src/main/java/com/jezhumble/javasysmon/NullMonitor.java

```diff
@@ -26,6 +26,8 @@ public MemoryStats physical() {
         return null;
     }
 
+    public MemoryStats physicalWithBuffersAndCached(){ return null; }
```

Comment:
We can reformat this method, so it will not be one-line. Just to follow the format of other methods in this class.